pages/system/carousel.js

```diff
@@ -0,0 +1,231 @@
+import * as React from "react";
+import * as Constants from "~/common/constants";
+import * as System from "~/components/system";
+import CodeBlock from "~/components/system/CodeBlock";
```

Comment:
Nit: Missing a whitespace between to keep align with other files, I'll correct this once I merge so don't worry about it.